backend-rust/src/migrations.rs

```diff
@@ -236,12 +236,14 @@ pub enum SchemaVersion {
     AmountSchedule,
     #[display("0022:Fix corrupted passive delegators")]
     FixCorruptedPassiveDelegators,
+    #[display("0023:Fix transaction count")]
```

Comment:
This schema version is only relevant for the scheme of the database and not the API, so there is no need to introduce this